src/Groups/homomorphisms.jl

```diff
@@ -729,15 +729,16 @@ function isomorphism(::Type{T}, A::FinGenAbGroup) where T <: GAPGroup
      A_to_A2 = inv(A2_to_A)
      # Create an isomorphic GAP group whose `GAPWrap.GeneratorsOfGroup`
      # consists of independent elements of the orders in `exponents`.
-     # (We cannot guarantee that these generators form a pcgs in the case
-     # `T == PcGroup`, hence we cannot call `abelian_group(T, exponents)`.)
      if T == PcGroup
+       # We cannot guarantee that these generators form a pcgs in the case
+       # `T == PcGroup`, hence we cannot call `abelian_group(T, exponents)`.
        if 0 in exponents
          GapG = GAP.Globals.AbelianPcpGroup(length(exponents), GapObj(exponents; recursive = true))
+         G = PcGroup(GapG)
```

Comment:
For `Pcp` groups the given exponents actually do correspond 1-1 to the given exponent vector.

On the other hand `IndependentGeneratorsOfAbelianGroup` output looks horrible:
```gap
gap> G:=AbelianPcpGroup([3,0,3,9,15]);
Pcp-group with orders [ 3, 0, 3, 9, 15 ]
gap> gens:=GeneratorsOfGroup(G);
[ g1, g2, g3, g4, g5 ]
gap> List(gens, Order);
[ 3, infinity, 3, 9, 15 ]
gap> IndependentGeneratorsOfAbelianGroup(G);
[ g2*g3^2*g4, g1, g3*g5^5, g4^6*g5^10, g5^6, g4^5*g5^10 ]
```

So I think we should avoid `IndependentGeneratorsOfAbelianGroup` and `IndependentGeneratorExponents` for pcp groups.